mmdet/models/anchor_heads/atss_head.py

```diff
@@ -444,7 +452,8 @@ def atss_target_single(self,
         num_valid_anchors = anchors.shape[0]
         bbox_targets = torch.zeros_like(anchors)
         bbox_weights = torch.zeros_like(anchors)
-        labels = anchors.new_zeros(num_valid_anchors, dtype=torch.long)
+        labels = anchors.new_zeros(
+            num_valid_anchors, dtype=torch.long) + background_label
```

Comment:
We could change it to `labels = anchors.new_empty(num_valid_anchors, dtype=torch.long).fill_(background_label)`.